packages/patternfly-3/react-console/package.json

```diff
@@ -38,6 +38,7 @@
     "classnames": "^2.2.5",
     "patternfly": "^3.52.1",
     "patternfly-react": "^2.10.0",
+    "react-bootstrap": "^0.32.1",
```

Comment:
maybe this should be peer-dependency like in pf-react? I would assume that consuming projects will have their own decency on pf-react (and react-bootstrap?)

Reply:
Right, thanks.